test/runtests.py

```diff
@@ -18,14 +19,14 @@
 Samples:
 
 test all folders:
-    {0}
+    runtests.py
 
 test only Array:
-    {0} Array
+    runtests.py Array
 
 test a single file:
-    {0} Basics/hello.js
-'''.format(sys.argv[0]))
+    runtests.py Basics/hello.js
```

Comment:
Alternatively can use basename(sys.argv[0]) to avoid hardcode the script name.

Currently tab completion doesn't work well with test scripts, because there are 2 scripts with similar file names -- runtests.sh and runtests.py. May we consider rename this one to `test.py`?